recipes/libfyaml/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set build_number = 0 %}
```

Comment:
```suggestion
```

Reply:
is this not considered best practice? I find it convenient to have everything to change when updating a package at the top of the recipe.

Reply:
The autoupdate bot will do most of the changes for you once this is merge and too much jinja for vatiables that are used only once make the recipe hard to read. So you can say that it is not considered the best pratice.